generators/entity-server/templates/dotnetcore/src/Project.Infrastructure/Data/ApplicationDatabaseContext.cs.ejs

```diff
@@ -144,15 +142,27 @@ namespace <%= namespace %>.Infrastructure.Data {
             });
             relationships.forEach(relationship => {
                 if (relationship.relationshipType === 'many-to-many') {
+                    // executed when using the entity generator to add relationships (jhipster entity entityName)
                     if (relationship.ownerSide) {
                         let ownerRelationshipFieldName = pascalizedEntityClass;
-                        let nonOwnerRelationshipFieldName = relationship.otherEntityNamePascalized; _%>
-            builder.Entity<<%= relationship.joinEntityNamePascalized %>>()
-                .HasKey(t => new { t.<%= ownerRelationshipFieldName %>Id, t.<%= nonOwnerRelationshipFieldName %>Id });
+                        let nonOwnerRelationshipFieldName = relationship.otherEntityNamePascalized; 
+                        let relationshipNamePascalized = toPascalCase(relationship.relationshipName);
```

Comment:
The varible is not define here ? (https://github.com/jhipster/jhipster-dotnetcore/blob/main/generators/entity/index.js#L119) 
if not can you declare here once

Reply:
Now using the variable defined at generators/entity/index.js